script.js

```diff
@@ -0,0 +1,368 @@
+    if (element[0] === 'rShift') {
+      document.querySelector('.rShift').after(clearfix);
+    }
+  });
+
+  if (shiftCase === false && sessionStorage.lang === 'ENG') {
+    arrValues.forEach((element) => {
+      let a = document.querySelector(`.${element[0]}`);
+      if (element.length < 2) {
+        a.innerText = element[0];
+      } else if (element[0] === 'enter') {
+      } else if (element[1] !== undefined) {
+        a.innerText = element[1];
+      }
+    });
+  }
+
+  if (shiftCase === true && sessionStorage.lang === 'ENG') {
+    arrValues.forEach((element) => {
+      let a = document.querySelector(`.${element[0]}`);
+      if (element.length < 2) {
+        a.innerText = element[0];
+      } else if (element[0] === 'enter') {
+      } else if (element[2] !== undefined) {
+        a.innerText = element[2];
+      }
+    });
+  }
+
+  if (shiftCase === false && sessionStorage.lang === 'RUS') {
+    arrValues.forEach((element) => {
+      let a = document.querySelector(`.${element[0]}`);
+      if (element.length < 2) {
+        a.innerText = element[0];
+      } else if (element[0] === 'enter') {
+      } else if (element[3] !== undefined) {
+        a.innerText = element[3];
+      }
+    });
+  }
+
+  if (shiftCase === true && sessionStorage.lang === 'RUS') {
+    arrValues.forEach((element) => {
+      let a = document.querySelector(`.${element[0]}`);
+      if (element.length < 2) {
+        a.innerText = element[0];
+      } else if (element[0] === 'enter') {
+      } else if (element[4] !== undefined) {
+        a.innerText = element[4];
+      }
+    });
+  }
+}
+
+keyboard.addEventListener('mousedown', function (event) {
+  textarea.focus();
+  if (event.target.classList[0] == 'keyboard') return;
+  lastPressButton = event.target;
+  setStyle(event.target, true);
+  identifyKey(event.target.dataset, 'down');
+  showKey();
+});
+
+keyboard.addEventListener('mouseup', function (event) {
+  textarea.focus();
+  if (event.target.dataset.code === 'CapsLock') return;
+  removeLastPressButtonStyle();
+  identifyKey(event.target.dataset, 'up');
+  showKey();
+});
+
+document.addEventListener('keydown', function (event) {
+  if (event.repeat) return;
+  event.preventDefault();
+  setStyle(document.querySelector(`.${keyboardElements[event.code][0]}`), true);
+  identifyKey(event, 'down');
+  checkLanguage();
+  showKey();
+});
+
+document.addEventListener('keyup', function (event) {
+  if (event.repeat) return;
+  if (event.key == 'CapsLock') return;
+  setStyle(
+    document.querySelector(`.${keyboardElements[event.code][0]}`),
+    false
+  );
+  identifyKey(event, 'up');
+  checkLanguage();
+  showKey();
+});
+
+function setStyle(code, bool) {
+  if (bool) code.classList.add('active');
+  else code.classList.remove('active');
+}
+
+function removeLastPressButtonStyle() {
+  lastPressButton.classList.remove('active');
+}
+
+function identifyKey(event, updown) {
+  if (event.code === 'ShiftLeft') {
```

Comment:
Function is too big and do a lot of different stuff. Better to brake it in separate functions and give them proper names. Single responsibility principle is also true for functions.